### PR TITLE
AZP: Fix release pipeline

### PR DESCRIPTION
**Pull request by avildema**
Fix release pipeline

### CODE REVIEW

**Review by yosefe — APPROVED** [review]
pls update to cuda 11_1 and PR to v1.10.x as well so 1.10 release will be fixed

File: buildlib/az-distro-release.yml [hunk 1]
```diff
@@ -30,9 +30,9 @@ jobs:
         ubuntu18_cuda10_2:
           build_container: ubuntu18_cuda10_2
           artifact_name: ucx-${{ replace(variables['Build.SourceBranch'], 'refs/tags/', '') }}-ubuntu18.04-mofed5.0-cuda10.2.deb
-        ubuntu18_cuda11.0:
-          build_container: ubuntu18_cuda11.0
-          artifact_name: ucx-${{ replace(variables['Build.SourceBranch'], 'refs/tags/', '') }}-ubuntu18.04-mofed5.0-cuda11.0.deb
+        ubuntu18_cuda11_1:
+          build_container: ubuntu18_cuda11_1
+          artifact_name: ucx-${{ replace(variables['Build.SourceBranch'], 'refs/tags/', '') }}-ubuntu18.04-mofed5.1-cuda11.1.deb
```
Comment (yosefe) [inline buildlib/az-distro-release.yml:35]: mofed version - let's revert to 5.0?

Comment (avildema) [reply buildlib/az-distro-release.yml:35]: ok

Comment (yosefe) [reply buildlib/az-distro-release.yml:35]: 5.0

**Review by yosefe — APPROVED** [review]
pls squash

**Pull request merged**